app/views/posts/_form.tmhl.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with(model: [@user, @post], url: user_posts_path(@user)) do |form| %>
+    <div class="mb-3">
+        <%= form.text_field :title, class: 'form-control', placeholder: 'Title' %>
+    </div>
+
+    <div class="mb-3">
+        <%= form.text_area :text, class: 'form-control', placeholder: 'Text' %>
+    </div>
+
+    <%= form.submit "Create Post", class: 'btn btn-primary' %>
+<% end %>
```

Comment:
- [ ] When I try to access the pages for the form, I get this error:

![image](https://github.com/NalianAchei2020/RubyonRails-blog/assets/77106962/059a8b4e-a6cb-43eb-bc14-40fb58e8f20f)

---
You need to name the file extension properly. The file should be named _**app/views/posts/_form.html.erb**_ and not _**app/views/posts/_form.tmhl.erb**_. The file extension should be _**html**_ and not _**tmhl**_. 

Kindly correct this. 👍 💻